doc/release-notes.md

```diff
@@ -101,6 +99,7 @@ Dash Core tree 0.12.1.x was a fork of Bitcoin Core tree 0.12.
 
 These release are considered obsolete. Old release notes can be found here:
 
+- [v0.14.0.1](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-0.14.0.1.md) released May/31/2019
```

Comment:
have the old release notes been copied here yet? I'm not finding it

Reply:
They have been copied, but they are not in `master` yet. You can find them here: https://github.com/dashpay/dash/blob/v0.14.0.x/doc/release-notes/dash/release-notes-0.14.0.1.md

Reply:
ahh, I was looking in develop